app/models/miq_action.rb

```diff
@@ -848,6 +848,31 @@ def action_container_image_analyze(action, rec, inputs)
     rec.scan
   end
 
+  def action_container_image_scan_all(action, rec, _inputs)
```

Comment:
> Since I do not have ems targeted policies, I raise the event against one image of each provider and scan all of it's sibling under the provider.

Just saw this comment